packages/cfd/src/Containers/cfd-password-modal.tsx

```diff
@@ -675,22 +675,20 @@ const CFDPasswordModal = ({
     const is_password_reset = error_type === 'PasswordReset';
     const [is_sent_email_modal_open, setIsSentEmailModalOpen] = React.useState(false);
 
-    const { poi_verified_for_bvi_labuan, poi_verified_for_vanuatu_maltainvest, poa_verified, manual_status } =
+    const { poi_verified_for_bvi_labuan_vanuatu, poi_verified_for_maltainvest, poa_verified, manual_status } =
         getAuthenticationStatusInfo(account_status);
 
     const [is_selected_mt5_verified, setIsSelectedMT5Verified] = React.useState(false);
 
     const getVerificationStatus = () => {
         if (jurisdiction_selected_shortcode === 'svg') {
             setIsSelectedMT5Verified(true);
-        } else if (jurisdiction_selected_shortcode === 'bvi') {
-            setIsSelectedMT5Verified(poi_verified_for_bvi_labuan);
-        } else if (jurisdiction_selected_shortcode === 'vanuatu') {
-            setIsSelectedMT5Verified(poi_verified_for_vanuatu_maltainvest);
+        } else if (jurisdiction_selected_shortcode === 'bvi' || jurisdiction_selected_shortcode === 'vanuatu') {
```

Comment:
can we change this to ['bvi','vanuatu'].includes(jurisdiction_selected_shortcode)